addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -90,6 +90,24 @@ onkyo:onkyoAVR:avr-livingroom [ipAddress="192.168.1.100", port=60128, volumeLimi
 
 Binding then automatically scale the volume level in both directions (100% = 50 = 100%).
 
+You can also change the way volume scaling works. This can be necessary if your receiver uses a different scaling system than 0-100.
```

Comment:
Can you add a new line after a sentence. It provides better maintainability.

Reply:
Done